inst/include/interface/rcpp/rcpp_objects/rcpp_interface_base.hpp

```diff
@@ -27,7 +28,8 @@ class Parameter {
  public:
  static uint32_t id_g; /**< global id of the parameter */
     uint32_t id_m; /**< id of the parameter */
-  double value_m; /**< initial value of the parameter */
+  double value_m = 0.0; /**< initial value of the parameter */
```

Comment:
I think the names are somewhat confusing since estimated_m doesn't always return an estimated value. Should these be initial and final instead?

Reply:
Renamed to final_value_m and initial_value_m in 4d18ff6c